components/page.jsx

```diff
@@ -12,6 +12,20 @@ var DevRibbon = (process.env.NODE_ENV === 'production' &&
                 : require('./dev-ribbon.jsx');
 
 var Page = React.createClass({
+  statics: {
+    handlerForPage: function(router, url) {
+      return router.match(url).routes[1].handler;
+    },
+    titleForHandler: function(handler) {
+      var title = 'Mozilla Learning';
+
+      if (handler.pageTitle) {
+        title = handler.pageTitle + ' - ' + title;
```

Comment:
I'm not sure if the convention is to put general site title first or title of the subpage.  But Webmaker seems to be doing this the same way so I guess this is the preferred way.